packages/chain-agnostic-permission/src/scope/constants.ts

```diff
@@ -89,3 +89,10 @@ export const KnownNotifications: Record<NonWalletKnownCaipNamespace, string[]> =
     bip122: [],
     solana: [],
   };
+
+/**
+ * Session properties for known CAIP namespaces.
+ */
+export enum KnownSessionProperties {
+  SolanaAccountChangedNotifications = 'solana_accountChanged_notifications',
```

Comment:
hmm... is it better to state what features I want or what client I am (and let the wallet infer what features i want)?.. Unsure

Reply:
🤔 I would think feature based subscriptions since it lets clients decide what bundles they prefer?

Reply:
i agree in general, but was wondering if it was different in this particular case where we don't intend for this to be used by any client but our own. Ehh, this is fine